### ISSUE TITLE
Videos that worked before are suddenly giving a "media not available" error

### ISSUE THREAD

**Issue by SuperrSonic** [jaw20/Crunchyroll-XML-Decoder#12]
Can someone confirm this?
Trying to download http://www.crunchyroll.com/galaxy-express-999/episode-26-the-skeletons-song-512670

I downloaded this video about a year ago, now even on the old builds it fails. Might be related to it being max 480p and hardsubbed? Kanokon is also failing.

**Comment by shinji257** [comment]
Are you still seeing the error?

I'll check this out when I get a minute over the weekend and see if my local patch fixed it.  I had fixed an issue related to videos that had no subs attached to them.

**Comment by SuperrSonic** [comment]
Yes, still the same message. There are a few things I found out, I'm not sure how relevant they might be.
The video in the op when I downloaded it a long time ago, as well as the others were using 2-pass and had the same bitrate, they were all ~128 MB.

Using another program to download the video failed, but some users said they had no problem dl it. Someone suggests logging in, I don't see the point, I then log in and it didn't work. On a whim I make a new account, log in and can now download those videos again. However they come out as CRF 19.

So I'm still not sure what the cause is, and why logging in mattered. Suffice it to say logging in on the XML Decoder did not change the error.

**Comment by shinji257** [comment]
Just an FYI.  I finally got around to testing this.  I do not get an error when retrieving this episode either via a guest account or as a logged in user.

**Comment by Dokman** [comment]
i am having it too, i will give you a log later
